accelerator/message.h

```diff
@@ -37,6 +37,7 @@ typedef enum ta_cli_arg_value_e {
   MWM_CLI,
   SEED_CLI,
   CACHE,
+  CONFIG,
```

Comment:
I am afraid of the existence of `CONFIG` which might be too general to understand. Also, it might cause potential conflicts.

Reply:
How about `CONF_FILE` ?
@jserv @ender503

Reply:
Something like `CONF_CLI` might be more consistent.